components/SearchDropdown/index.tsx

```diff
@@ -0,0 +1,86 @@
+import styles from "../../styles/SearchDropdown/dropdown.module.css";
```

Comment:
Can you also lint all the files by running

```
npm run prettify
```

Reply:
Sure